fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -61,15 +62,55 @@ def test_create_get_and_delete_container(self):
             self.assertRaises(e)
             self.logger.debug("%s container was successfully removed from the object storage" % containerName)
 
-    def test_create_container(self):
+    def test_create_container_with_object_deleting_them(self):
         """
-        Test if it can be possible create a new container into the object storage.
+        Test if it can be possible create a new container into the object storage, upload a text file, download it and remove the
+        object and the container.
         """
 
         suffix = datetime.utcnow().strftime('%Y%m%d%H%M%S%m')
         containerName = TEST_CONTAINER_PREFIX + suffix
+        textObjectName = TEST_TEXT_OBJECT_PREFIX + suffix
 
         response = self.swift_operations.create_container(containerName)
         self.assertIsNone(response)
         self.test_world['containers'].append(containerName)
-        self.logger.debug("Created %s container was created" % containerName)
\ No newline at end of file
+        self.logger.debug("Created %s container was created" % containerName)
+
+        ## Uploading the object
+
+        response = self.swift_operations.create_text_object(containerName, SWIFT_RESOURCES_PATH
+                                                            + TEST_TEXT_OBJECT_PREFIX + TEXT_EXTENSION,
+                                                            textObjectName + TEXT_EXTENSION)
+        origin = hashlib.md5(open(SWIFT_RESOURCES_PATH + TEST_TEXT_OBJECT_PREFIX + TEXT_EXTENSION, 'rb')
+                          .read()).hexdigest()
+
+        self.assertIsNone(response)
```

Comment:
Add a message error for each assert, in order to see a comprehensible reason of problems in result reports.